### PR TITLE
[ondemand] fix docker ssh-agent

### PR DESCRIPTION
**Pull request by jjergus**

### CODE REVIEW

File: aws/ondemand/main.sh [hunk 1]
```diff
@@ -24,6 +24,7 @@ ok
 log "Initial system configuration..."
 chmod a-x /etc/update-motd.d/*
 mkdir -p /home/ubuntu/.cache
+chown ubuntu:ubuntu /home/ubuntu/.cache
```
Comment (fredemmott) [inline aws/ondemand/main.sh]: is this related?

Comment (jjergus) [reply aws/ondemand/main.sh]: oops not related, but I noticed random commands failing because they can't write there so I fixed that as well
